ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py

```diff
@@ -301,3 +301,42 @@ def test_eye_like(
         device=on_device,
         ground_truth_backend=ground_truth_backend,
     )
+
+
+@handle_test(
+    fn_tree="functional.ivy.experimental.sequence_empty",
+    shape=helpers.get_shape(
+        allow_none=False,
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    dtype=helpers.get_dtypes("numeric", full=False),
+    container_flags=st.just([False]),
+    test_instance_method=st.just(False),
+    test_gradients=st.just(False),
+)
+def test_sequence_empty(
+    *,
+    shape,
+    dtype,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+):
+    ret = helpers.test_function(
```

Comment:
Oh, and here
`ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py:330:5: F841 local variable 'ret' is assigned to but never used`